Gopkg.toml

```diff
@@ -69,6 +77,10 @@
   name = "k8s.io/client-go"
   version = "kubernetes-1.15.4"
 
+[[override]]
+  source = "gopkg.in/fsnotify/fsnotify.v1"
+  name = "gopkg.in/fsnotify.v1"
```

Comment:
Something is not right with the CAPI dependencies. Stevo figured this fixes the issues we had with `dep`.